packages/common/src/utils/formatting.ts

```diff
@@ -58,7 +58,9 @@ export const currencies = [
 ];
 
 export const formatBoolean = <T>(v: T) =>
-    ['True', 'true', 'yes', 'Yes', '1', 'T'].includes(`${v}`) ? 'Yes' : 'No';
+    ['True', 'true', 'yes', 'Yes', '1', 'T'].includes(`${v}`)
```

Comment:
I believe you need to update tests in: `/packages/common/src/utils/formatting.test.ts`

Not sure if we need to update docs but didn't really see a reference to `yes` on our codebase!

Reply:
Thanks, good call!